src/components/Checkbox/Checkbox.tsx

```diff
@@ -1,117 +1,79 @@
 import clsx from 'clsx';
-import React, {
-  useRef,
-  useEffect,
-  ChangeEventHandler,
-  MutableRefObject,
-} from 'react';
+import React from 'react';
+import { useUID } from 'react-uid';
 import styles from './Checkbox.module.css';
-import { ENTER_KEYCODE } from '../../util/keycodes';
+import CheckboxInput from '../CheckboxInput';
+import type { CheckboxInputProps } from '../CheckboxInput';
+import CheckboxLabel from '../CheckboxLabel';
+import type { CheckboxLabelProps } from '../CheckboxLabel';
```

Comment:
nit, could be
```suggestion
import CheckboxInput, { CheckboxInputProps } from '../CheckboxInput';
import CheckboxLabel, { CheckboxLabelProps } from '../CheckboxLabel';
```

though if you want to be more explicit that's fine

Reply:
Will condense imports, thanks